src/main/java/com/back/global/standard/util/Ut.kt

```
@@ -0,0 +1,75 @@
+        }
+    }
+
+    object json {
+        lateinit var objectMapper: ObjectMapper
+
+        @JvmStatic
+        @JvmOverloads
+        fun toString(obj: Any, defaultValue: String = ""): String {
+            return try {
+                objectMapper.writeValueAsString(obj)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)